### DISCUSSION TITLE
Download of jpg's

### DISCUSSION THREAD

**Discussion by wg25** [jomjol/AI-on-the-edge-device#2137] [General] [answered]
Hi, is it possible to download the stored jpg's over WiFi?

**Comment by caco3** [comment, marked as answer]
Yes and no.
The static images are accessible through the file server, eg.  `http://<IP>/fileserver/config/ref1.jpg`

The cam image is only stored in RAM to keep the sd-card write low. You can access it using `http://<IP>/img_tmp/alg.jpg`, see https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/#img_tmpalgjpg

**Reply by wg25** [reply]
Thanks for the hint. And great job by the way!!!
